src/hw_isolation_record/manager.cpp

```diff
@@ -185,7 +185,28 @@ std::optional<sdbusplus::message::object_path> Manager::createEntry(
                           _bus, entryObjPath, *this, recordId, severity,
                           resolved, associationDeftoHw, entityPath)));
 
-        utils::setEnabledProperty(_bus, isolatedHardware, resolved);
+        // TODO:https://github.com/ibm-openbmc/openpower-hw-isolation/issues/39
```

Comment:
Can we not handle only the "Connection Timeout Exception"? if we hit some other error means, what should happen? still, we need to allow? and do we need a PEL in this case?

and the DIMM is managed by the Inventory manager so the expectation is the same?

and I am surprised that the "Set DBus call" alone failed because we will **read** something from the PLDM service before coming here in this flow.

Reply:
DIMM/Core we do not want to miss on guarding as the system will be working on a faulty hardware.

We have stated during restart/refresh of service we will re-read and set the property so users are going to live with this issue (GUI not being set)

Regarding PEL, we can create one if you think so.